pkg/generator/templates/cloud-init-ubuntu.template

```diff
@@ -42,6 +41,7 @@ write_files:
 {{- end -}}
 {{- end }}
 runcmd:
+- until apt-get update -qq && apt-get install --no-upgrade -qqy docker.io socat nfs-common logrotate jq policykit-1; do sleep 1; done
```

Comment:
`apt_update: true` and  `apt-get update -qq` is the same.

To install packages use the following syntax. 
```
packages:
    - pwgen
    - pastebinit
    - [libpython2.7, 2.7.3-0ubuntu3.1]
```